### ISSUE TITLE
wrong target for mandate of heaven

### ISSUE THREAD

**Issue by Lopatouovalil** [Savolainen5/The-Heirs-to-Aquitania#5]
i got event "Mandate of Heaven" as Netherlands, which happens when you became GP as Chinese country.

**Comment by Savolainen5** [comment]
Yeah, I just posted on this on Discord last night. Not sure why it's firing. I incorporated a fix which I haven't pushed yet, but I haven't checked the fix, either.

**Comment by Lopatouovalil** [comment]
and other thing, do you plan to adjust migration to colonies because some countries basically make countries(for example englaland)from colonies in few years.

**Comment by Savolainen5** [comment]
So you're saying some European countries have TOO much migration to their colonies?

**Comment by Lopatouovalil** [comment]
yes, in some cases
